web/src/lib/elements/ui/dataTable/data-table.svelte

```diff
@@ -266,7 +268,7 @@
             >
               {#each row.cells as cell (cell.id)}
                 <Subscribe attrs={cell.attrs()} let:attrs>
-                  <Table.Cell {...attrs}>
+                  <Table.Cell class="mammt" {...attrs}>
```

Comment:
WTH is "mammt"?

Reply:
I added a random class to see to which HTML element that cell would correspond and I forgot to remove it lol